app/celery/scheduled_tasks.py

```diff
@@ -165,30 +172,30 @@ def check_job_status():
     from jobs
     where job_status == 'in progress'
     and template_type in ('sms', 'email')
-    and scheduled_at or created_at is older that 120 minutes.
+    and scheduled_at or created_at is older than 60 minutes.
     if any results then
         raise error
         process the rows in the csv that are missing (in another task) just do the check here.
     """
-    minutes_ago_120 = datetime.utcnow() - timedelta(minutes=120)
-    minutes_ago_125 = datetime.utcnow() - timedelta(minutes=125)
+    minutes_ago_30 = datetime.utcnow() - timedelta(minutes=30)
+    minutes_ago_35 = datetime.utcnow() - timedelta(minutes=35)
```

Comment:
Can we go with 60 and 65 minutes here?

Reply:
I reverted this from 120 minutes back to 30. Originally this was 30 minutes and we bumped it up to [60](https://github.com/cds-snc/notification-api/commit/db955f3038742b641af5997cd2a1ed1e96bd9520) then to [120](https://github.com/cds-snc/notification-api/commit/6f71d29a8c60dcadee9562ea624d0c34171a41e9) as bandaid fixes for the issue this PR is solving.

Reply:
Okay makes sense! 👍